kobo/apps/mass_emails/user_queries.py

```diff
@@ -1,13 +1,58 @@
+from datetime import timedelta
 from math import inf
 
-from django.db.models import QuerySet
+from django.db.models import Q, QuerySet
+from django.utils.timezone import now
 
 from kobo.apps.kobo_auth.shortcuts import User
 from kobo.apps.organizations.models import Organization
+from kobo.apps.openrosa.apps.logger.models import Instance
 from kobo.apps.stripe.utils import get_organization_plan_limits
+from kpi.models import Asset
 from kpi.utils.usage_calculator import get_storage_usage_by_user_id
 
 
+def get_inactive_users(days: int = 365) -> QuerySet:
+    """
+    Retrieve users who have been inactive for a specified number of days.
+
+    A user is considered inactive if:
+    - They have not logged in within the given period (or never logged in).
```

Comment:
Can we update this comment to note that users who were recently created will not be considered inactive, even if they haven't logged in yet?